sw/device/silicon_creator/lib/log.h

```diff
@@ -0,0 +1,39 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_LOG_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_LOG_H_
+#include <stdarg.h>
+
+#include "sw/device/silicon_creator/lib/error.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * Prints a status message to UART0, formatted according to the format
+ * string `format`.
+ *
+ * This function only supports the format specifiers required by the
+ * mask ROM:
+ * - %s, which prints a NUL-terminated string.
```

Comment:
s/NUL/NULL/

Reply:
Went for lowercase `null-terminated` to match the wikipedia page. I think it was originally spelt NUL because that is the name of the ASCII character.